pkg/server/registry/apigroups/acorn/apps/events.go

```diff
@@ -80,7 +80,7 @@ func (s *eventRecordingStrategy) Create(ctx context.Context, obj types.Object) (
 		},
 		Type:        AppCreateEventType,
 		Severity:    apiv1.EventSeverityInfo,
-		Details:     details,
+		Details:     &details,
```

Comment:
Curious, what's the necessity here? Did we change how Details works?

Reply:
`Mapify` now returns a struct and all usages of `GenericMap` on the API types are now pointers.